scripts/asm-installer/install_asm

```diff
@@ -1539,6 +1552,21 @@ configure_package() {
   if [[ -n "${_CI_ASM_IMAGE_TAG}" ]]; then
     run kpt cfg set asm anthos.servicemesh.tag "${_CI_ASM_IMAGE_TAG}"
   fi
+
+  if [[ "${CA}" == "mesh_ca" && -n "${TRUSTED_GCP_PROJECTS}" ]]; then
+    # Gather the trust domain aliases from projects.
+    TRUST_DOMAIN_ALIASES="${PROJECT_ID}.svc.id.goog"
+    while read -d ',' -r trusted_gcp_project; do
+      TRUST_DOMAIN_ALIASES="${TRUST_DOMAIN_ALIASES} ${trusted_gcp_project}.svc.id.goog"
+    done <<EOF
+${TRUSTED_GCP_PROJECTS}
+EOF
+    run eval kpt cfg set asm anthos.servicemesh.trustDomainAliases "${TRUST_DOMAIN_ALIASES}"
```

Comment:
Why is this eval necessary?

Reply:
let me add comment to explain.

Command like kpt treat the word in quotes as a single word, while kpt need ${TRUST_DOMAIN_ALIASES} to be splitting words for alias array. If we remove quotes, the lint will complain.

`eval` will translate the quoted var into words to workaround both.